packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -145,13 +190,21 @@ export class SchedulerModel {
                     updated_at: new Date(),
                 })
                 .returning('*');
-            const targetPromises = newScheduler.targets.map(async (target) =>
-                trx(SchedulerSlackTargetTableName).insert({
+            const targetPromises = newScheduler.targets.map(async (target) => {
+                const isSlack = 'channel' in target;
```

Comment:
Let's use this logic in a util class to identify the slack target type. Like `isUpdateSchedulerSlackTarget()`